src/app/features/tech-record/components/vehicle-technical-record/vehicle-technical-record.component.ts

```diff
@@ -106,7 +108,32 @@ export class VehicleTechnicalRecordComponent implements OnInit, AfterViewInit {
     }
   }
 
-  isAnyFormInvalid(forms: Array<CustomFormGroup | CustomFormArray>) {
+  getVehicleDescription(techRecord: TechRecordModel, vehicleType: VehicleTypes | undefined): string {
+    switch (vehicleType) {
+      case VehicleTypes.TRL:
+        return techRecord.vehicleConfiguration ?? '';
+      case VehicleTypes.PSV:
+        return techRecord.bodyMake && techRecord.bodyModel ? `${techRecord.bodyMake}-${techRecord.bodyModel}` : '';
+      case VehicleTypes.HGV:
+        return techRecord.make && techRecord.model ? `${techRecord.make}-${techRecord.model}` : '';
+      default:
+        return 'Unknown Vehicle Type';
+    }
+  }
+
+  createTest(isComplete?: string): void {
+    if (isComplete) {
+      this.router.navigate(['test-records/create-test/type'], { relativeTo: this.route });
+    } else {
+      alert(
```

Comment:
Should it be alert or should we just add a red banner box at the top of the page?

Reply:
A bit like when you try to submit amends on a tech record when you're missing fields?

Reply:
It should be an alert, that's what they want. Check the ticket.

Reply:
But that SD is from an app, might be worth just asking Matt

Reply:
Just got confirmation from Matt. We're going ahead with this.